sass/admin_console_base/_sys_css_variables.scss

```diff
@@ -1,6 +1,7 @@
 :root {
     // Cloned default hex values from base/css_variables to be used in the system console
     --sys-away-indicator: #ffbc1f;
+    --sys-away-indicator-black-opacity: #997113;
```

Comment:
why is this called `*-black-opactiy`? Doesn't this lead to confusion?

Reply:
haha, for sure it will lead to confusion 😕 , you know the hardest part is naming, maybe @michaelgamble can help us to come up with a better name for this color. This is the one used for the warnings chips color text, that is basically the away-indicator with a black opacity as Michale mentioned in the loom video.

Reply:
maybe something like `--sys-away-indicator-dark`?

Reply:
that one receives a +1 from me, that one was on the top of my list for naming this. Let's wait for @michaelgamble opinion.